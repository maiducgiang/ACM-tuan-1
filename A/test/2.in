4
C G
G C
B H
H B