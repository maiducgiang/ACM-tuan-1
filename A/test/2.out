C => D => H => G
G => F => B => C
B => C => D => H
H => G => F => B
